alignment.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>
+        Alignment
+    </title>
+    <link rel="stylesheet" href="css/alignment.css">
+    <style>
+        .system-fonts {
```

Comment:
I don't think you need this style here - we don't really need it anywhere aside from in the `index.html` since that's the only place it's necessary.